apps/frontend/src/network/routes.ts

```diff
@@ -0,0 +1 @@
+export const TEAMS_PATH = 'teams';
```

Comment:
BTW I've only *started* with these, we can gradually use for more and more routes...